blog/2024/09/juliacon-2025-announcement.md

```diff
@@ -0,0 +1,102 @@
+
+If this program interests you, please fill out the following form: <https://forms.gle/rNjqoXGgQ3TnpcRf8>
+
+### JuliaCon Locals
+
+We had over a dozen community members reach out to talk about hosting a JuliaCon Local at JuliaCon 2024! 
+The first step in this process is always to start a local meetup. We’re happy to help with that as much as we can, you can reach out on the #community channel on Slack at any time. 
+
+Once you have an established meetup and you’d like to host a JuliaCon Local please reach out to [raye@julialang.org](mailto:raye@julialang.org) or [ranjan@julialang.org](mailto:ranjan@julialang.org) for more information.
+
+We are working on a unified playbook for how to build your local community, how to get access to awesome speakers, and how to find spaces and funds for your meetup, so please keep an eye out for that on our usual channels!
+
+## JuliaCon Global 2024 Wrap Up
+
+JuliaCon 2024 was a wonderful, record breaking event for our community. If you were able to attend in person we hope you enjoyed the conference and plan to attend future JuliaCons! If you were an online viewer or have been watching our talks as they've been uploaded on YouTube hopefully you plan to join us in 2025!
+
+![JuliaCon Crowd](/assets/blog/2024-JuliaCon/140A0016.jpg)
+
+### Julia Community Prizes
+
+We want to be sure you didn’t miss the Julia Community Prizes. These prizes were announced at the closing session of JuliaCon 2024 on July 12th, but if you missed it, the awardees this year were:
+
+* **The JuMP Team**: Including **Oscar Dowson**, **Benoît Legat**, and **Miles Lubin**, for their long-term commitment to JuMP.jl and the broader mathematical programming ecosystem in Julia.
+
+* **The Makie Team:** Including **Simon Danisch**, **Julius Krumbiegel**, **Frederic Freyer**, and **Anshul Singhvi**, for their creation of Makie.jl and improvements to the state of data visualization in Julia.
+
+* **Guillaume Dalle:** For his various contributions to the Julia ecosystem including DifferentiationInterface.jl and ModernJuliaWorkflows.
+
+Awardees are nominated by the Julia community, and then winners are selected by a committee of senior community members based on their holistic contributions across the ecosystem.
+
+### JuliaCon 2024 Organizers and Volunteers
+
+JuliaCon wouldn’t happen without an amazing group of organizers and volunteers. The organizing committee put in an incredible amount of work over the past year, and an incredible 70+ volunteers made the conference possible.
+
+I first want to highlight the incredible Local Organizing Committee. Led by Gareth Thomas, who is unmatched in his ability to bring people together for a common cause, the local committee was the key to both JuliaCon Local Eindhoven and JuliaCon Global 2024!
```

Comment:
The rest of the article is written in plural style (also because the author is "JuliaCon Organizing Committee"), I guess this should also be plural
```suggestion
We first want to highlight the incredible Local Organizing Committee. Led by Gareth Thomas, who is unmatched in his ability to bring people together for a common cause, the local committee was the key to both JuliaCon Local Eindhoven and JuliaCon Global 2024!
```